README.md

```diff
@@ -1,17 +1,100 @@
 # Conda Recipes
 
-Example recipes for the new conda build system.  Use
+Community-maintained repository of conda package build recipes
+for 592+ conda packages.
```

Comment:
I don't agree with putting a specific number here but it'll get outdated quite quickly